raycicmd/converter.go

```diff
@@ -181,6 +181,22 @@ func (c *converter) convertPipelineGroup(g *pipelineGroup) (
 	}
 
 	for _, step := range g.Steps {
+		// filter steps by tags
+		if step_tags, ok := step["tags"]; ok && tagFilters != nil {
+			skip := true
+			for _, tag := range step_tags.([]interface{}) {
+				for _, filter := range tagFilters {
+					if tag == filter {
+						skip = false
+					}
+				}
```

Comment:
could you build a `map` of tags, at the beginning of the function, and do a map check here (rather than doing a loop)?